modules/emotions.json

```diff
@@ -0,0 +1,2373 @@
+{

```

Comment:
I have never seen so many colours in my life :laughing: 

This should be in a config file as this deviates from the current design pattern.